### ISSUE TITLE
PhosphoRS score without Annotations

### ISSUE THREAD

**Issue by ypriverol** [compomics/compomics-utilities#21]
Hi @mvaudel do you think would be possible to implement phosphoRS without the fragment annotations. That means if I give you a peptide sequence, spectrum, modification, and positions. ?

**Comment by mvaudel** [comment]
Yes, it means a bit of refactoring but this is also a feature @david-bouyssie asked for, hopefully we will have the time to implement it during his visit in July.
